src/mobile/dropdown/test/className.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, fixture } from "@open-wc/testing";
+import { MobileDropdown } from "../index";
+
+describe("MobileDropdown", () => {
+  describe("className", () => {
+    it("className default prop is null", async () => {
+      const container = new MobileDropdown();
+      const el = await fixture(container);
+      expect(el.classList.length).to.be.equal(0);
+    });
+
+    it("className default prop set successfully", async () => {
+      const container = new MobileDropdown({ className: "options-class" });
+      const el = await fixture(container);
+      expect(el.classList.length).to.be.equal(1);
+      expect(el.className).to.have.equal("options-class");
+    });
+
+    it("className prop replace successfully'", async () => {
```

Comment:
@nmanhit 
Please remove `className` and change the `title` to the same content with expectation.
Ex: `it("it should be replace by \"replace-class\" when changing by setter")`